packages/components/src/components/relative-datepicker/relative-datepicker.scss

```diff
@@ -0,0 +1,18 @@
+#dc-relative-datepicker {
+    width: 100%;
+    text-align: center;
+    padding-bottom: 16px;
+
+    span {
+        color: var(--purchase-main-2);
+        font-size: var(--text-size-xs);
+        font-weight: bold;
+        line-height: 1.43;
+        cursor: pointer;
+    }
+    // This css is for make date input field invisible
+    input {
```

Comment:
It is better to follow BEM for this. Add a class name like `dc-relative-datepicker__input` is easier to maintain in the future.

Reply:
Sure